src/main/java/org/sagebionetworks/bridge/sdk/models/subpopulations/ConsentStatus.java

```diff
@@ -0,0 +1,110 @@
+package org.sagebionetworks.bridge.sdk.models.subpopulations;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sagebionetworks.bridge.sdk.utils.Utilities.TO_STRING_STYLE;
+
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class ConsentStatus {
+
+    public static boolean isUserConsented(Map<SubpopulationGuid,ConsentStatus> statuses) {
+        checkNotNull(statuses);
+        if (statuses.isEmpty()) {
+            return false;
+        }
+        for (ConsentStatus status : statuses.values()) {
+            if (status.isRequired() && !status.isConsented()) {
+                return false;
+            }
+        }
+        return true;
+    }
```

Comment:
Tests cover the empty vs. state of consents, testing various combinations of status objects.